src/main/webapp/index.jsp

```diff
@@ -6,9 +6,17 @@
 
 <html>
     <head>
-        <title> CS-KMITL: Group 1</title>
+        <link rel="stylesheet" type="text/css" href="css/tab.css">
+        <title>หน้าหลัก</title>
     </head>
     <body>
-        <h1><c:out value = "${requestScope.message}" /></h1>
+        <H3>ใบกำกับภาษีอิเล็กทรอนิกส์</H3>
+        <button class="btn" onclick="createTaxInvoice()" >ออกใบกำกับภาษี</button>
     </body>
 </html>
+
+<script>
+function createTaxInvoice(){
+    location.href = 'createTaxInvoice.jsp';
+}
```

Comment:
Will it break when JS is disable ?

Reply:
Yes , it breaks.
Should I change it to href?